content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md

```diff
@@ -36,6 +36,7 @@ Authentication is not enabled by default for kubectl and Helm installations. In
         # custom max body size for file uploading like backing image uploading
         nginx.ingress.kubernetes.io/proxy-body-size: 10000m
     spec:
+      ingressClassName: nginx
```

Comment:
Please apply this change to the example output as well so that they align.

Reply:
what is the example output sorry?

Reply:
https://github.com/mrksph/website/blob/patch-1/content/docs/1.6.3/deploy/accessing-the-ui/longhorn-ingress.md?plain=1#L79-L104